src/main/java/mil/dds/anet/utils/Utils.java

```diff
@@ -336,4 +345,60 @@ public static boolean isEmailWhitelisted(String email, List<String> whitelistDom
 
     return isWhitelistedEmail || isValidWildcardDomain;
   }
+
+  /**
+   * Resizes an image
+   * 
+   * @param imageBase64 The image as a Base64 string
+   * @param width The desired output width
+   * @param height The desired output height
+   * @param imageFormatName The desired output format (png, jpg)
+   * @return The resized image as a Base64 string
+   * @throws IOException When the conversion fails
+   */
+  public static String resizeImageBase64(String imageBase64, int width, int height,
+      String imageFormatName) throws IOException {
+
+    if (imageBase64 == null || imageBase64.isEmpty()) {
+      return imageBase64;
+    }
+
+    // From Base64-string to BufferedImage
+    final BufferedImage imageBytes = convert(imageBase64);
```

Comment:
Maybe we can make this method take a binary image as argument, to avoid decoding/reencoding base64 -> binary and back. We could encode to base64 in the last moment when returning the result

Reply:
I am not following you on this one. There is a string available with the image data in Base64. To resize it I need to convert it to a BufferedImage, resize it and convert it back to Base64. I do not see how I can go around this.